pyproject.toml

```diff
@@ -22,22 +22,24 @@ exclude = ["tests/"]
 [tool.poetry.dependencies]
 python = "^3.7"
 Django = "^3.0"
-PyJWT = "^2.1.0"
-strawberry-graphql = "^0.67.1"
+PyJWT = "^1.5"
```

Comment:
I'd probably stick with the latest version available (v1.7.1) of pyjwt prior to the next major release (2.0.0).